RAKE/RAKE.py

```diff
@@ -122,21 +160,27 @@ def generate_candidate_keyword_scores(phrase_list, word_score):
 
 
 class Rake(object):
-    def __init__(self, stop_words):
+        
```

Comment:
flake8 is just failing here because there's some whitespace or something, I can fix it up post merge so don't worry about it